crates/uv/tests/it/pip_tree.rs

```diff
@@ -1159,3 +1159,76 @@ fn outdated() {
     "###
     );
 }
+
+/// Test that dependencies with multiple marker-specific requirements
+/// are only displayed once in the tree.
+#[test]
+#[cfg(feature = "pypi")]
+fn no_duplicate_dependencies_with_markers() {
+    let context = TestContext::new("3.12").with_exclude_newer("2026-01-01T00:00:00Z");
+
+    let requirements_txt = context.temp_dir.child("requirements.txt");
+    requirements_txt.write_str("pylint").unwrap();
```

Comment:
We might want to add a local package, e.g., to `scripts/packages` which has repeated `Requires-Dist` values. It's _okay_ to use pylint, but it'll make the test slower. We can track this in a separate issue if you prefer.

Reply:
I've refactored the test to create the package inline, which seems to be much simpler than persisting multiple packages to disk!